PocketKit/Sources/PocketKit/Home/HomeViewController.swift

```diff
@@ -190,10 +178,26 @@ extension HomeViewController {
                 cell.saveButton.addAction(uiAction, for: .primaryActionTriggered)
             }
 
-            if let action = model.reportAction(for: item, at: indexPath), let uiAction = UIAction(action) {
-                cell.overflowButton.addAction(uiAction, for: .primaryActionTriggered)
+            let menuActions = model.overflowActions(for: item, at: indexPath)
+            cell.overflowButton.menu = UIMenu(children: menuActions.compactMap(UIAction.init))
+    
+            return cell
+        case .recommendationCarousel(let objectID):
+            let cell: HomeCarouselItemCell = collectionView.dequeueCell(for: indexPath)
+            
+            guard let viewModel = model.recommendationCarouselViewModel(for: objectID) else {
+                cell.model = nil
+                return cell
+            }
+            
+            cell.model = viewModel
+            
+            if let action = model.saveAction(for: item, at: indexPath), let uiAction = UIAction(action) {
+                cell.saveButton.addAction(uiAction, for: .primaryActionTriggered)
```

Comment:
Same idea here, let's set the save button's action in the view's `reconfigure()` method. Ideally, setting the cell's `model` property should be the only thing we need to do in order for the view to be fully configured.